Sources/ArgumentParser/Parsable Properties/OptionGroup.swift

```diff
@@ -111,3 +129,15 @@ extension OptionGroup {
     self.init(visibility: .default)
   }
 }
+
+// MARK: Deprecated
+
+extension OptionGroup {
+  @_disfavoredOverload
+  @available(*, deprecated, renamed: "init(title:visibility:)")
```

Comment:
Do we need this still, SAP isn't ABI stable and the above initializer is source compatible?

Reply:
eh, well very very minimally source breaking if someone ever used a point free version of the function: `OptionGroup.init(visibility:)`I really doubt this is the case though.